x-pack/plugins/siem/public/containers/kpi_network/index.tsx

```diff
@@ -29,6 +30,20 @@ export interface KpiNetworkProps extends QueryTemplateProps {
   children: (args: KpiNetworkArgs) => React.ReactNode;
 }
 
+const formatHistogramData = (
```

Comment:
nit - this logic can be in the server side

Reply:
Moved this to `elasticsearch_adapter`, and yes it simplified my data types and logic a lot, thank you, @XavierM !